pkg/processor/trigger/nats/jetstream/types.go

```diff
@@ -48,7 +75,107 @@ func NewConfiguration(id string,
 		return nil, errors.Wrap(err, "Failed to decode attributes")
 	}
 
-	// TODO: validate
+	err = newConfiguration.PopulateConfigurationFromAnnotations([]trigger.AnnotationConfigField{
+		{Key: "nuclio.io/nats-allow-reconnect", ValueBool: &newConfiguration.AllowReconnect},
```

Comment:
```suggestion
		{Key: "nuclio.io/nats-jetstream-allow-reconnect", ValueBool: &newConfiguration.AllowReconnect},
```
and same for all below to avoid confusion with normal nats